tests/ctst/features/iam-policies/AssumeRole.feature

```diff
@@ -0,0 +1,99 @@
+Feature: IAM Policies for Assume Role Session Users
+  This feature allows you to create and attach IAM policies for IAM users.
+  IAM users should have the permissions to perform the actions that they are granted in their IAM policies.
+
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-AssumeRole
+  Scenario Outline: Assume Role User is not authorized to perform the actions with no IAM policy attached to the role
+    Given an existing bucket "" "without" versioning
+    And an object "" that "<objectExists>" "without" version-Id
```

Comment:
do we really need the "objectExists" option? It is fine creating the file even if we don't use it, but having a "Given" that possibility do nothing feels a bit off IMHO; if this is really not needed, we could just split the scenario.